src/components/Error/Error.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type ErrorProps = {
+  hasError: boolean;
+  setHasError: (error: boolean) => void;
+  errorMessage: string;
+};
+
+export const Error: React.FC<ErrorProps> = ({
```

Comment:
Consider giving a more specific name to the component, such as 'ErrorNotification', to avoid confusion with built-in Error objects or other error components. 🤔

Reply:
totally agree